cli/compilers/js.rs

```diff
@@ -0,0 +1,24 @@
+use crate::compilers::CompiledModule;
+use crate::compilers::CompiledModuleFuture;
+use crate::file_fetcher::SourceFile;
+use crate::state::ThreadSafeState;
+use std::str;
+
+pub struct JsCompiler {}
+
+impl JsCompiler {
+  pub fn compile_async(
```

Comment:
All of these `compile_async` methods don't seem to do anything async. It's passed a `source_file`, which already contains the data. It seems to just wrap it up in a FutureResult... is there a more appropriate name for this method?